### PR TITLE
fix(discord): Fix alert formatting

### LINKED ISSUE #57289: Discord metric alert not formatted correctly
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Setup metric alerts in discord
2. Get notification

### Expected Result

No need for a second link below the title. It should look like the slack notification. 

Image

### Actual Result

Image

### Product Area

Alerts

### Link

_No response_

### DSN

_No response_

### Version

_No response_

### PR DESCRIPTION
**Pull request by 17hogeju**
Closes https://github.com/getsentry/sentry/issues/57289

Screenshot 2023-10-02 at 9 12 37 AM

### CODE REVIEW

**Comment by ceorourke** [comment]
Is there a test that hits this path?

**Comment by 17hogeju** [comment]
> Is there a test that hits this path?

Yes, test_message_builder.py. I will look again and see if it needs updating.

File: tests/sentry/integrations/discord/test_message_builder.py [hunk 1]
```diff
@@ -30,12 +38,13 @@ def test_metric_alert_without_incidents(self):
...
             )
         )
+
         assert DiscordMetricAlertMessageBuilder(self.alert_rule).build() == {
             "content": "",
             "embeds": [
                 {
                     "title": title,
-                    "description": f"<{link}|*{title}*>  \n",
+                    "description": "",
```
Comment (ceorourke) [inline tests/sentry/integrations/discord/test_message_builder.py:47]: What's the difference here between `test_metric_alert_without_incidents` and `test_metric_alert_with_selected_incident`? Like when/how would a metric alert fire that has no incidents?

Comment (17hogeju) [reply tests/sentry/integrations/discord/test_message_builder.py:40]: I based my tests off of tests/sentry/integrations/slack/test_message_builder.py. It looks like `test_metric_alert_without_incidents` is to mean a metric alert with no active incident and never triggered while `test_metric_alert_with_selected_incident`  means a metric alert with an active incident

https://github.com/getsentry/sentry/pull/34128#issuecomment-1115407930

I am not aware of the context of when a metric alert would need to fire without incidents...

Comment (ceorourke) [reply tests/sentry/integrations/discord/test_message_builder.py:40]: Yeah that's odd 🤔

**Review by ceorourke — APPROVED** [review]

**Pull request merged**